src/index.js

```diff
@@ -1,12 +1,12 @@
+import './index.css';
```

Comment:
Global styles should always be included before component styles. CSS is cascading, and components are expected to be the highest priority styling, so you don't want the global index styles to override them (which they were when imported last)